template/module/wizards/wizard_model_view.xml

```diff
@@ -5,7 +5,7 @@
 <openerp>
 <data>
 
-<record id="wizard_model_view" model="ir.ui.view">
+<record id="wizard_model_view_form" model="ir.ui.view">
```

Comment:
This one should be also `model_name_view_form`. The model name determines if it's a transient model (wizard) or not.